package.json

```diff
@@ -11,8 +11,8 @@
   },
   "license": "UNLICENSED",
   "author": "REI",
-  "main": "dist/src/bin/main.mjs",
-  "bin": "dist/src/bin/main.mjs",
+  "main": "dist/bin/main.mjs",
```

Comment:
Removing the import assertion allows `tsc` to output to `dist/` instead of `dist/src`. See this thread:
https://stackoverflow.com/a/58941798